web/src/main/java/com/microsoft/applicationinsights/web/extensibility/modules/WebSessionTrackingTelemetryModule.java

```diff
@@ -183,98 +145,17 @@ public boolean getIsUserModuleEnabled() {
         return isUserModuleEnabled;
     }
 
-    /**
-     * Gets a value indicating whether new sessions should be generated.
-     * @return True if new sessions should be generated, false otherwise.
-     */
-    public boolean getGenerateNewSessions() {
-        return generateNewSessions;
-    }
-
-    /**
-     * Gets the session timeout in minutes, or null if not set yet.
-     * @return Session timeout in minutes
-     */
-    public Integer getSessionTimeoutInMinutes() {
-        return sessionTimeoutInMinutes;
-    }
-
     // endregion Public
 
     // region Private
 
     private void parseArguments(Map<String, String> argumentsMap) {
```

Comment:
I think that this entire code can be removed (need to also check how we initiate the class when loading configuration)